File: xFit/xFit.Model/Klijent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace xFit.Model
{
	public class Klijent
	{
		public int KlijentId { get; set; }

		public string? Ime { get; set; }

		public string? Prezime { get; set; }

		public DateTime? DatumRodjenja { get; set; }

		public int? KorisnikId { get; set; }

	}
}

File: xFit/UI/xfit_admin/lib/models/klijent.dart
import 'package:json_annotation/json_annotation.dart';

part 'klijent.g.dart';

@JsonSerializable()
class Klijent{
	int? klijentId; 
  String? ime;
  String? prezime;
  DateTime? datumRodjenja;
  int? korisnikId;
  
  Klijent(this.klijentId,this.ime,this.prezime,this.datumRodjenja,this.korisnikId);

  factory Klijent.fromJson(Map<String, dynamic> json) => _$KlijentFromJson(json);

  Map<String, dynamic> toJson() => _$KlijentToJson(this);

}
